core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -169,7 +169,7 @@ sorttable = {
     for (var i=0; i<table.tBodies[0].rows.length; i++) {
       text = sorttable.getInnerText(table.tBodies[0].rows[i].cells[column]);
       if (text != '') {
-        if (text.match(/^-?[�$�]?[\d,.]+%?$/)) {
+        if (text.match(/^-?[£$¤]?[\d,.]+%?$/)) {
```

Comment:
Heh, github says this line changed but I don't see it. I'm worried it's a difference in some unprintable character that will throw this off. Any way to make sure this line really didn't change? copy and paste from the previous version or is that what you did?